### PR TITLE
restore buildings to zoom 13

### PR DESCRIPTION
**Pull request by nvkelso**
- [x] Update tile selections &/or transforms
- [x] Update tests
- [x] Data migrations required
- [ ] No doc updates required (fix regression over prod)

### CODE REVIEW

**Comment by nvkelso** [comment]
@zerebubuth I added a tight data migration just for z13 features (ignoring edge cases for underground and no – those should still be calculated but its a 1% expense).

File: integration-test/845-buildings-z13.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Earlier work in 845 dropped buildings from zoom 13
+
+# http://www.openstreetmap.org/way/23654700
+# http://www.openstreetmap.org/way/60500069
+assert_has_feature(
+    13, 1310, 3170, 'buildings', 'kind': 'building'})
```
Comment (zerebubuth) [inline integration-test/845-buildings-z13.py]: Syntax error here - you need a `{` to begin the tags `dict` literal.

**Comment by zerebubuth** [comment]
This seems to have broken the following tests:

* integration-test/653-unify-building-part.py
* integration-test/834-park-building.py
* integration-test/842-normalize-building-kind.py

Could you check to make sure that the new behaviour is what you want, and that these aren't new regressions, please?

File: integration-test/845-buildings-z13.py (hunk 1 above)
Comment (nvkelso) [inline integration-test/845-buildings-z13.py]: Thanks! Fixed in 8d54e2955c4dc4f1a95dbe0001b162f70f1d2da0.

File: data/migrations/v1.0.0-polygon.sql [hunk 2]
```diff
@@ -27,6 +27,12 @@ UPDATE
     AND COALESCE(mz_landuse_min_zoom, 999) <> COALESCE(mz_calculate_min_zoom_landuse(planet_osm_polygon.*), 999);
 
 UPDATE planet_osm_polygon
+  SET mz_building_min_zoom = mz_calculate_min_zoom_buildings(planet_osm_polygon.*)
+  WHERE mz_calculate_min_zoom_buildings(planet_osm_polygon.*) IS NOT NULL;
```
Comment (rmarianski) [inline data/migrations/v1.0.0-polygon.sql]: trailing semicolon, probably want an `AND` here instead?

Comment (nvkelso) [inline data/migrations/v1.0.0-polygon.sql]: Fixed in 990e58ee9bc19e33cc758c40556ca7a71cfc14ec (by removing most of that WHERE line).

**Comment by rmarianski** [comment]
LGTM :+1:

**Comment by nvkelso** [comment]
I think I've fixed the z13 building problem, but am still working on the 3 other integration test failures @zerebubuth called out in https://github.com/tilezen/vector-datasource/pull/1036#issuecomment-246328811.

**Comment by nvkelso** [comment]
Rockin' it, @rmarianski!

**Comment by rmarianski** [comment]
:guitar:
